### ISSUE TITLE
Content Manager - Compilation error: `minizip/unzip.h: No such file or directory`

### ISSUE THREAD

**Issue by tsarquis88** [wazuh/wazuh#20100]
## Description

After #19915, the minizip library was added as a dependency to the `shared_modules/utils` tools. These changes brought a new issue while compiling the content manager module:

```
/home/runner/work/wazuh/wazuh/src/shared_modules/utils/zlibHelper.hpp:17:10: fatal error: minizip/unzip.h: No such file or directory
   17 | #include "minizip/unzip.h"
      |          ^~~~~~~~~~~~~~~~~
```

The error in actions can be seen here: [vulnerability-scanner-modules](https://github.com/wazuh/wazuh/actions/runs/6782027992/job/18433492931?pr=20085).

> Note: In that PR, the vulnerability scanner RTR was not executed because the only changed files were under `utils/`.

## DoD

- [x] Fix compilation error.
- [x] Compliance with:
  - [x] [RTR](https://github.com/wazuh/wazuh/blob/dev-14153-vulndet-refactor/src/wazuh_modules/vulnerability_scanner/doc/rtr.md)
  - [x] [GitHub Actions](https://github.com/wazuh/wazuh/actions/workflows/vulnerability-scanner-tests.yml)
- [ ] Add or Update any relevant documentation to reflect the changes made.
